### PR TITLE
Update hammer.dm

### PR DESCRIPTION
**Pull request by Chevalt**
Description : 
A simple change to the charge hammer, making it suitable for the belt slot, considering the foreman's hammer and homewrecker could already do that.

## Changelog
tweak: tweaked a few things
fix: fixed a few things

### CODE REVIEW

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 1]
```diff
@@ -140,7 +140,7 @@
 	switched_on_qualities = list(QUALITY_HAMMERING = 60)
 	switched_off_qualities = list(QUALITY_HAMMERING = 35)
 	toggleable = TRUE
-	slot_flags = SLOT_BACK
+	slot_flags = SLOT_BELT|SLOT_BACK
```
Comment (ThePainkiller) [inline code/game/objects/items/weapons/tools/hammer.dm:143]:
```suggestion
	slot_flags = SLOT_BELT | SLOT_BACK
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
This should make it compile.

File: code/game/objects/items/weapons/tools/hammer.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/game/objects/items/weapons/tools/hammer.dm:143]: Oh wait, it works just as fine, nvm feel free to ignore

**Comment by king5327** [comment]
I'd personally say nah because the rocket hammer has the whole 'charge' feature too, but I don't really use it so I can't say.

**Comment by Kazkin** [comment]
I'd say its fitting for it, personally, the rocket hammer is a junk weapon and it makes semi-sense it can't fit the belt. If someone makes a Pr I don't mind giving all 2h sledges belt slot AND suit slot.

**Pull request merged**